website/src/app/kb/client-apps/linux-gui-client/readme.mdx

```diff
@@ -61,26 +61,22 @@ already work properly.
 
 1. Download the RPM:
    [Download the latest Linux GUI `.rpm` from GitHub Releases](https://www.github.com/firezone/firezone/releases)
-2. `sudo dnf install systemd-resolved` (Installing it explicitly prevents it
+1. `sudo dnf install systemd-resolved` (Installing it explicitly prevents it
    from being auto-removed if Firezone is removed)
-3. `sudo dnf install ./firezone-client-gui-*.rpm`
-4. `sudo usermod -aG firezone-client $USER`
-5. `sudo systemctl enable firezone-client-tunnel.service` (See
-   https://www.freedesktop.org/software/systemd/man/latest/systemd.preset.html,
-   "It is not recommended to ship preset files within the respective software
-   packages implementing the units". The Fedora family of distros also seem to
-   have their own policy that installing a service should not auto-start or
-   enable it.)
```

Comment:
I've dropped this for consistency with our Debian package. Our service just idles anyway unless a GUI connects so this doesn't seem like a big problem to me and will make things easier for most users.